src/document.c

```diff
@@ -473,8 +491,8 @@ FIELD_PREPROCESSOR(geoPreprocessor) {
 
   char *end1 = NULL, *end2 = NULL;
   double lon = strtod(c, &end1);
-  double lat = strtod(pos, &end2);
-  if (*end1 || *end2) {
+  double lat = strtod(pos, &end2); /// failing here
+  if (*end1 || (*end2 && *end2 != '"')) {
```

Comment:
So basically I will be able to write something like this : `"1,2"foo`?
And why we suddenly decided that it's ok to get a string with `"`?

Reply:
B/c ReJSON keeps strings with `"`.